test/unit_tests/braket/experimental/autoqasm/test_api.py

```diff
@@ -792,3 +792,129 @@ def main() -> None:
 
     with pytest.raises(errors.InconsistentNumQubits):
         main()
+
+
+def test_unnamed_retval_python_type() -> None:
```

Comment:
There are many very similar tests in `test_types.py`, shall we move these there?

Reply:
Yeah, that does seem like a better place for these.